frontend/templates/cucumber/tests/browser/package.json.hbs

```diff
@@ -1,12 +1,9 @@
 {
   "name": "ipv-cri-{{ kebabCase criName }}-front-tests",
-  "version": "1.0.0",
```

Comment:
These changes are not part of the "setup npm workspaces for lambda repos" so should be pulled out into a separate PR.